nautobot_golden_config/nornir_plays/config_intended.py

```diff
@@ -85,6 +103,7 @@ def run_template(  # pylint: disable=too-many-arguments
         output_file_location=output_file_location,
         default_drivers_mapping=get_dispatcher(),
         jinja_filters=jinja_env.filters,
+        jinja_env=jinja_env,
```

Comment:
Have you confirmed this is working? I am not sure I follow how this would work, based on following how this works I would expect this to cause a stacktrace based on a mismatched function signature. https://github.com/nautobot/nornir-nautobot/blob/5f1236ed5c1e2ac33fc980c65c4c9b360b7e25d1/nornir_nautobot/plugins/tasks/dispatcher/default.py#L157-L165

Reply:
Correct. I submitted 2 PRs related to this feature. The other is to add the additional parameter you highlighted.

https://github.com/nautobot/nornir-nautobot/pull/97

Reply:
ok, marking as draft, since this would need to 

1. Be applied to the nornir-nautobot
2. update the packaging here